plugins/source/aws/resources/services/ecr/repository_image_scan_findings.go

```diff
@@ -17,49 +17,88 @@ func repositoryImageScanFindings() *schema.Table {
 		Name:        "aws_ecr_repository_image_scan_findings",
 		Description: `https://docs.aws.amazon.com/AmazonECR/latest/APIReference/API_ImageScanFindings.html`,
 		Resolver:    fetchEcrRepositoryImageScanFindings,
-		Transform:   transformers.TransformWithStruct(&models.ImageScanWrapper{}),
+		Transform: transformers.TransformWithStruct(&ecr.DescribeImageScanFindingsOutput{},
+			transformers.WithSkipFields("ImageId", "NextToken", "ResultMetadata"),
```

Comment:
Why skip `ImageId`

Reply:
[`imageId`](https://docs.aws.amazon.com/AmazonECR/latest/APIReference/API_DescribeImageScanFindings.html#ECR-DescribeImageScanFindings-response-imageId) field is an object of [`ImageIdentifier`](https://docs.aws.amazon.com/AmazonECR/latest/APIReference/API_ImageIdentifier.html) type. It will be a JSOn column (that we already have info about). We can leave this in the response, though, WDYT?

Reply:
I see no reason to drop data that is returned from the API. Lets try and stay as close as possible to the API, in the past when we have deviated, that is when issues have arisen especially when one user wants it one way and another wants it in a different way

Reply:
c4b934974e37b8028ed8810966f05feac440ff8c